engine/src/flutter/display_list/dl_text.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_DISPLAY_LIST_DL_TEXT_H_
+#define FLUTTER_DISPLAY_LIST_DL_TEXT_H_
+
+#include "flutter/display_list/geometry/dl_geometry_types.h"
+#include "third_party/skia/include/core/SkRefCnt.h"
```

Comment:
I think we can change the getter to SkTextBlob* and we won't need SkRefCnt included here...?